daft/runners/ray_runner.py

```diff
@@ -1244,6 +1240,16 @@ def __init__(
                     address,
                 )
         else:
+            if address is not None:
+                if not address.endswith("10001"):
+                    warnings.warn(
+                        f"The address to a Ray client server is typically at port :10001, but instead we found: {address}"
+                    )
+                elif address.startswith("ray://"):
```

Comment:
shouldn't this be negative?
```python
elif not address.startswith("ray://"):
```

Reply:
and shouldn't this just be it's own condition? this makes more sense to me:
```python
if not address.endswith("10001"):
    warnings.warn(
        f"The address to a Ray client server is typically at port :10001, but instead we found: {address}"
    )
if not address.startswith("ray://"):
    warnings.warn(
        f"Expected Ray address to start with 'ray://' protocol but found: {address}. Automatically prefixing your address with the protocol to make a Ray connection: ray://{address}"
    )
    address = "ray://" + address
```

Reply:
Dang yes how did this go through